src/common/system_signal_handling.c

```diff
@@ -133,11 +146,41 @@ static void _dt_sigsegv_handler(int param)
 #endif
 
 static int _times_handlers_were_set = 0;
+
+#if defined(_WIN32)
+void dt_set_unhandled_exception_handler_win()
+{
+  ExcHndlInit();
+
+  if(1 == _times_handlers_were_set)
+  {
+    gchar *name_used;
+    int fout;
+    BOOL ok;
+
+    if((fout = g_file_open_tmp("darktable_bt_XXXXXX.txt", &name_used, NULL)) == -1)
+      fout = STDOUT_FILENO; // just print everything to stdout
+
+    if(fout != STDOUT_FILENO) close(fout);
+
+    // Set up logfile name
+    ok = ExcHndlSetLogFileNameA(name_used);
```

Comment:
So does anything bad happen if this is called second time?
If not, i think we should just do it.
Reason: gm already does some dumb shit with signal handlers. even if it currently does not call `ExcHndlSetLogFileNameA()` by itself, there is absolutely zero guarantee that it will not do so in the future...

Othervise replace `if(...) ...` with `if(!...) return;   ...`